controllers/skills.js

```diff
@@ -0,0 +1,105 @@
+const skills = require("../models/skills");
+const logger = require("../utils/logger");
+
+/**
+ * Award a skill
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - User object
+ * @param res {Object} - Express response object
+ */
+async function awardSkill(req, res) {
+  let addedData;
+
+  try {
+    const { username } = req.params;
+    addedData = await skills.awardSkill(req.body, username);
+  } catch (error) {
+    logger.error("Error posting skill data: ", error);
+    return res.boom.notFound("User doesn't exist");
+  }
+
+  return res.json({
+    message: "Added data successfully!",
+    content: addedData,
+  });
```

Comment:
Why it is outside the try block ?

Reply:
I had some lint errors for no-return-value that's why I moved those return statements outside block with let scope.
I'll make the change